.github/workflows/codecov.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     name: Generate code coverage
     needs: diff
     if: github.event.pull_request.draft == false && needs.diff.outputs.isRust == 'true'
-    runs-on: [self-hosted, ubuntu]
+    runs-on: [self-hosted, self-hosted-ubuntu]
     strategy:
       fail-fast: true
     steps:
```

Comment:
I can't comment on the line, but you'll need to comment out line 36-38 in this file to turn off the caching.

Reply:
I'll simplify this one and we'll just migrate the codecov for now.  I'll remove those lines.